recipes/miniwdl/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "miniwdl" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "fbb646a09af08f0d13f2f197eba01db576a70745190fa8546d50d165f21682d4"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
Windows is failing because you need to define entry points, see these links:
 - https://github.com/chanzuckerberg/miniwdl/blob/master/setup.cfg#L24
 - https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=selectors#python-entry-points